docs/prod-host-setup.md

```diff
@@ -77,8 +82,12 @@ for Firecracker processes that are unresponsive, and kills them, by SIGKILL.
 
 ## Jailer Configuration
 
-Using Jailer in a production Firecracker deployment is highly recommended,
-as it provides additional security boundaries for the microVM.
+For assuring secure isolation in production deployments, Firecracker should
+only be started only using the `jailer` binary that's part of each Firecracker
```

Comment:
```suggestion
must be started using the `jailer` binary that's part of each Firecracker
```

Reply:
Thanks for catching this typo. Now is resolved in a new iteration.